ui/src/components/VirtualKeyboard.tsx

```diff
@@ -26,6 +26,7 @@ const AttachIcon = ({ className }: { className?: string }) => {
 
 function KeyboardWrapper() {
   const [layoutName, setLayoutName] = useState("default");
+  const [depressedButtons, setDepressedButtons] = useState("");
```

Comment:
This tracks which buttons are "sticky down" to add them to the main keyboard's `depressed-key` class... so they get a different CSS style.